generate-dhparam.sh

```diff
@@ -1,7 +1,12 @@
 #!/bin/bash -e
 
-# The first argument is the bit depth of the dhparam, or 2048 if unspecified
-DHPARAM_BITS=${1:-2048}
+# The first argument is the bit depth of the dhparam, 1024 if unspecified and SSL_POLICY is set to Mozilla-Old,
+# or 2048 if unspecified and SSL_POLICY is set to any policy other than Mozilla-Old (including the default).
+if [[ $SSL_POLICY == "Mozilla-Old" ]]; then
```

Comment:
Technically `==` is not correct in Bash (or any other POSIX shell).  The equality operator is a single `=`, but `==` is widely tolerated.

Reply:
Will fix, thanks !